great_expectations/datasource/fluent/snowflake_datasource.py

```diff
@@ -482,6 +521,20 @@ def test_connection(self, test_assets: bool = True) -> None:
                     )
                 ) from e
             raise
+        if not self.schema_:
+            LOGGER.info("No schema provided, skipping schema presence check")
+            return None
+        if _is_bracketed_by_quotes(self.schema_):
+            warnings.warn(
+                message=f"Schema {self.schema_} is enclosed in double quotes and would fail sqlalchemy based schema check; skipping",
+                category=GxDatasourceWarning,
+            )
+            return None
```

Comment:
It's possible I can make this work for quoted identifiers, too, but I'd need to do some more thorough testing.
Since quoted identifiers are even less common for schemas, I'd like to punt on this extra handling up front.